core/io/resource.cpp

```diff
@@ -261,6 +263,11 @@ Ref<Resource> Resource::duplicate(bool p_subresources) const {
 		}
 		Variant p = get(E.name);
 
+		if (E == script_property) {
```

Comment:
```suggestion
		if (E.name == "script") {
```
As multiple properties with the same name are not allowed.

Reply:
Thank you for your feedback. Addressed.